test/test.js

```diff
@@ -160,3 +160,4 @@ tape('invalid tint', (t) => {
         t.end();
     });
 });
+
```

Comment:
oh nice @shabnomnom! can we add a test that uses a tint that [triggers these two conditionals](https://github.com/mapbox/makiwich/blob/expanding-brightness-threshholds-for-pin-colors/index.js)? You should be able to assert that that fill in the returned svg is black/white

Reply:
Sure thing. I added a test with the following orange color:
```
tinycolor("f58220").getBrightness()
153.213
```
this color is now dark and has white marker assigned to it